docs/ldml/tr35-keyboards.md

```diff
@@ -1384,13 +1420,13 @@ And a subsequent transform could transform the Z string, looking back (using @be
 
 ### 5.20 <a name="Element_reorder" href="#Element_reorder">Element: reorders, reorder</a>
 
-The reorder transform is applied after all transform except for those with `type="final"`.
+The reorder transform is applied after all transforms except for those with `type="final"`.
 
 This transform has the job of reordering sequences of characters that have been typed, from their typed order to the desired output order. The primary concern in this transform is to sort combining marks into their correct relative order after a base, as described in this section. The reorder transforms can be quite complex, keyboard layouts will almost always import them.
```

Comment:
> The reorder transforms can be quite complex, keyboard layouts will almost always import them.

Glad to see this sentence here.

That said, we may want to ensure it sticks out in some way if we expect users (i.e. keyboard authors) to reference this document with any regularity; with it being the last sentence in the paragraph, I could see people skimming past it easily.

If this is targeted more at developers of platforms that may look to implement support for LDML keyboards, it's probably fine as is.

Reply:
It's both, somewhat aimed at implementers first.  We have a number of comments about best practices, but definitely not a good on-ramp for authors. 

These don't strictly belong in the spec, but they could be adjacent documents:

- [ ] Start a draft guide for authors. 
- [ ] Start a draft guide for implementers.

Reply:
@jahorton  created tickets for the above here: -> https://unicode-org.atlassian.net/issues/?jql=issueKey%20in%20(CLDR-15847%2CCLDR-15848)